dev/tests/api-functional/testsuite/Magento/GraphQl/Customer/IsEmailAvailableTest.php

```diff
@@ -47,4 +47,55 @@ public function testEmailAvailable()
         self::assertArrayHasKey('is_email_available', $response['isEmailAvailable']);
         self::assertTrue($response['isEmailAvailable']['is_email_available']);
     }
+
+    /**
+     * @expectedException \Exception
+     * @expectedExceptionMessage GraphQL response contains errors: Email should be specified
```

Comment:
Looks like wrong error message expected.

Reply:
Thanks for that, I hadn't noticed the message change when merging in the latest 2.3-develop :+1: 

I've addressed this point, and the code style issues